sdk/tables/Azure.Data.Tables/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 3.0.0-beta.6 (Unreleased)
 
+### Fixed
```

Comment:
nit:  this should be "Key Bug Fixes" by [spec](https://azure.github.io/azure-sdk/policies_releasenotes.html)